srcpkgs/gcc/template

```diff
@@ -684,20 +685,13 @@ libsanitizer_package() {
 }
 
 libssp-devel_package() {
-	depends="libssp>=${_minorver}"
-	short_desc+=" - SSP (StackSmashingProtection) library - development files"
-	pkg_install() {
-		vmove "usr/lib/libssp*.a"
-		vmove usr/lib/gcc/${_triplet}/${_minorver}/include/ssp
-	}
+	short_desc+=" - (Transitional dummy package)"
+	build_style="meta"
 }
 
 libssp_package() {
-	short_desc+=" - SSP (StackSmashingProtection) library"
-	pkg_install() {
-		vmove "usr/lib/libssp.so*"
-		vlicense COPYING.RUNTIME RUNTIME.LIBRARY.EXCEPTION
-	}
+	short_desc+=" - (Transitional dummy package)"
+	build_style="meta"
 }
```

Comment:
Let's remove them all together, and let's `gcc` `replaces="libssp>=0 libssp-devel>=0"`